packages/@n8n/nodes-langchain/nodes/embeddings/EmbeddingsOpenAI/EmbeddingsOpenAi.node.ts

```diff
@@ -171,6 +171,11 @@ export class EmbeddingsOpenAi implements INodeType {
 						default: 'https://api.openai.com/v1',
 						description: 'Override the default base URL for the API',
 						type: 'string',
+						displayOptions: {
+							hide: {
+								'/@version': [1.2],
```

Comment:
Wouldn't this be better using the newer option to hide on any version > 1.2 so if there is a 1.3 we don't need to update this.

Same with the hides below as well.